packages/common/package.json

```diff
@@ -62,7 +62,7 @@
     "react": "^16.8.6",
     "react-icons": "^2.2.7",
     "react-input-autosize": "^2.2.1",
-    "react-router-dom": "^4.3.1",
+    "react-router-dom": "^5.0.1",
```

Comment:
oh boy, there's a router in common 🤔 curious if it clashes with the router in the app

Reply:
We should keep these in sync, but the reason why is we're building primitives around React Router's Link component. Don't think we actually have any route tables in Common.

Reply:
aah, okay. It might be worth looking into how to move that control to the app instead, it's not ideal for common components to have routing preferences

(not urgent/critical, but a good to have)